HousingSearchApi/V1/Interfaces/ElasticElasticSearchWrapper.cs

```diff
@@ -8,17 +8,17 @@
 
 namespace HousingSearchApi.V1.Interfaces
 {
-    public class ElasticElasticSearchWrapper : IElasticSearchWrapper
+    public class ElasticSearchWrapper : IElasticSearchWrapper
```

Comment:
should this class be inside interfaces?

Reply:
physically location do you mean? the directory location?